### ISSUE TITLE
Make it easier for verifiers to opt in or out of poor-quality records

### ISSUE THREAD

**Issue by kitenetter** [BiologicalRecordsCentre/iRecord#1290]
This is a recurring theme that has been raised again recently. Some verifiers would prefer not to be shown records that are of "poor quality". Other verifiers prefer to see everything and decide themselves if the record is worth following up for more details.

The definition of "poor quality" varies, but often refers to:
- Records with grid ref precision of worse than 10 km square
- Records of animals (usually invertebrates) lacking stage information
- Records of non-wild species (e.g. plants in gardens, a common problem with iNaturalist records)

I think we need to find a simple way to allow verifiers to opt in or out to low-precision grid refs from all sources.

I think the choice of blur on for sensitive records on iRecord should have a maximum blur of 10 km square; we should remove the 100 km blur.

I think we need to do more on iRecord itself to prevent invertebrate records being added without a stage term.

I'm not sure what can be done to improve stage handling in iNat, nor to filter out garden plants.

**Comment by japonicus** [comment]
From BSBI's perspective a 2km grid-ref precision threshold might be appropriate as records at worse precision have not been allowed in our database since 2020 (pre-2020 data are still acceptable at lower resolution).

I suspect that the garden plant issue is not resolvable fully, but it would be great to have an easy way for recorders to state that they are in a garden, (that's out-of-scope for this issue).

BSBI separately allows marking of records as 'not mappable as part of wild distribution' in parallel to the validation state, i.e. a record could be both 'confirmed' and 'do-not-map'. Something of that sort might provide verifiers with a quick and useful way of dealing with obvious garden records (and zoo animals). 'do-not-map' is also intended to avoid inappropriate rejection of correctly identified occurrences.

**Comment by johnvanbreda** [comment]
@kitenetter do we know if there is a field in the iNaturalist API data for indicating if a record is wild? If so we could capture that. There is a field for captive or cultivated observations so we could include that in the data we collect - it's possible that this is the field underlying the "organism is wild" flag but I'm not sure.

We could then add a quality tickbox filter to the top of the verification page. This would apply a predetermined filter -
* exclude non-wild/captive
* exclude >10km (tbc) precision (this isn't a blurring issue, as they see full precision, apart from for the iNat data)
* records of animals without stage

As a further enhancement, this could have a drop down arrow next to it which displays a panel with configuration for the above filters, i.e. which to apply and which precision level to require.

The quality tickbox could also be a tri-state control, i.e. good quality/bad quality/any quality.

I think the maximum blur level for sensitive records should be considered separately as it doesn't affect verification. There is an argument for having 100km blurs, when a record is of an organism where there are relatively few suitable locations (e.g. an eagle eyrie in islands), a 10km blur may give it away?

**Comment by cubitmg** [comment]
The api has a GET "observation_fields" request. However, there are many observation_fields that mention 'wild':
Is it wild?
Wild/Cultivated
Wild to Urban
Planted or wild?
Wild or cultivated?
Cultivated or Wild
Wild Caught
Fighting in wild
Captive C or Wild = W
Wild/Revegetated/Cultivated

Each of these have their own set of values .e.g: 
Wild plants
Plants cultivated WITHIN natural range
Plants cultivated OUTSIDE natural range Plant naturally established outside natural range
Invader
Cultivated exotic
Cultivated

There appear to be about 30,000 different Observation_fields and they can be created by anyone using this page: https://www.inaturalist.org/observation_fields/new.

I can't see there being any sensible way to retrieve this information even if it were recorded on a record. This wouldn't be an obvious thing to do for most users.
